src/main/java/com/flightstats/hub/util/HubUtils.java

```diff
@@ -43,6 +45,7 @@ public class HubUtils {
     private final Cluster hubCluster;
     private final ObjectMapper objectMapper;
     private final String uriScheme;
+    public static final String FILE_SYSTEM_SEPARATOR = FileSystems.getDefault().getSeparator();
```

Comment:
I am adding this constant in HubUtils as SpokePathUtils is not public class , due to that could not access outside package.